apps/dotcms-ui/src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -278,7 +280,17 @@ export class DotPageSelectorComponent implements ControlValueAccessor {
     }
 
     private cleanAndValidateQuery(query: string): string {
-        const cleanedQuery = this.cleanQuery(query);
+        let cleanedQuery = '';
+        if (this.isTwoStepSearch(query)) {
+            const url = this.parseUrl(query);
+            url.host = this.cleanHost(decodeURI(url.host));
+            url.pathname = this.cleanPath(url.pathname);
+            cleanedQuery = `//${url.host}/${url.pathname}`;
+        } else if (query.startsWith('//')) {
+            cleanedQuery = this.cleanHost(query);
+        } else {
+            cleanedQuery = this.cleanPath(query);
+        }
```

Comment:
We need two else?

Reply:
yea, first one checking if is a host, second one is a path, because is not a full search evaluated in the fist condition. ( if)